curriculum/challenges/english/10-coding-interview-prep/rosetta-code/vector-dot-product.md

```diff
@@ -7,57 +7,54 @@ forumTopicId: 302343
 
 # --description--
 
-A vector is defined as having three dimensions as being represented by an ordered collection of three numbers: (X, Y, Z).
+A vector can have one or more values represented by an ordered collection. Examples could be (x), (x, y), or (x, y, z).
 
 # --instructions--
 
-Write a function that takes any numbers of vectors (arrays) as input and computes their dot product. Your function should return `null` on invalid inputs such as vectors of different lengths.
+Write a function that takes two vectors (represented as one-dimensional arrays) as input and computes their dot product. Your function should return `null` on invalid inputs such as vectors of different lengths or passing anything other than two vectors.
 
 # --hints--
 
-dotProduct should be a function.
+`dotProduct` should be a function.
 
 ```js
 assert.equal(typeof dotProduct, 'function');
 ```
 
-dotProduct() should return null.
+`dotProduct()` should return `null`.
 
 ```js
 assert.equal(dotProduct(), null);
 ```
 
-dotProduct(\[[1], [1]]) should return 1.
+`dotProduct([1], [1])` should return `1`.
 
 ```js
 assert.equal(dotProduct([1], [1]), 1);
 ```
 
-dotProduct(\[[1], [1, 2]]) should return null.
+`dotProduct([[1], [1, 2]])` should return `null`.
```

Comment:
There should only be one pair of brackets per vector, so this line should be changed to

`dotProduct([1], [1, 2])` should return `null`.